attest/verifier/types/src/convert/collateral.rs

```diff
@@ -52,7 +52,7 @@ impl TryFrom<prost::Collateral> for Collateral {
         sgx_collateral.qe_identity_issuer_chain_size =
             qe_identity_issuer_chain.as_bytes().len() as u32;
 
-        let mut qe_identity = value.qe_identity;
+        let mut qe_identity = value.qe_identity.clone();
```

Comment:
tradeoff: A clone() that didn't exist before, but it's cheaper than caller having to clone the parent